packages/expo-video-thumbnails/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import { NativeModulesProxy } from '@unimodules/core';
+
+const { ExpoVideoThumbnails } = NativeModulesProxy;
```

Comment:
Would you please extract fetching `ExpoVideoThumbnails` "native module" to a separate file and provide a web implementation that would throw an error?